updatecli/updatecli.d/jdk11.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: Bump JDK11 version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  getLatestJDK11Version:
+    kind: githubrelease
+    name: Get the latest Adoptium JDK11 version
+    spec:
+      owner: "adoptium"
+      repository: "temurin11-binaries"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionfilter:
+        kind: regex
+        # jdk-11.0.12+7(https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.12%2B7) is OK
+        pattern: "^jdk-11.(\\d*).(\\d*).(\\d*)(.(\\d*))+(\\d*)$"
```

Comment:
Not sure this regexp is optimised/strict enough (I'm thinking about the unescaped `.` and `+`) but if it does the work it's not that important.

Reply:
I'm going to merge this PR, and open a subsequent with an improved regexp.